### PR TITLE
Fixes/import export

### PR DESCRIPTION
**Pull request by adrianherrera**
Fixes https://github.com/S2E/s2e-env/issues/228 and other issues associated with exporting custom projects.

There should really be some tests for this, but I'm not really sure how to write them.

### CODE REVIEW

File: s2e_env/commands/export_project.py [hunk 1]
```diff
@@ -81,75 +81,62 @@ def handle(self, *args, **options):
...
-            # Copy project scripts and config files and rewrite the S2E
-            # environment path in these files
+            # Do **not** export these files
+            blacklist = CONSTANTS['import_export']['blacklist']
+            if not options['export_results']:
+                blacklist.extend(['s2e-last', 's2e-out-*'])
+
+            # Copy the project directory
+            logger.info('Copying project %s', self.project_name)
+            shutil.copytree(self.project_path(), export_dir,
```
Comment (vitalych) [inline s2e_env/commands/export_project.py]: Is this going to resolve any symlinks in the folder?
The zip must not contain symlinks to files outside the project folder, these symlinks have to be turned into real files instead.

**Comment by vitalych** [comment]
It think that this can be easily tested by creating a new project (e.g., a driver project), exporting it, then importing back. And then compare if the files in the imported project are all there.

File: s2e_env/commands/export_project.py (hunk 1 above)
Comment (adrianherrera) [reply s2e_env/commands/export_project.py]: Yep. `copytree` has a `symlinks=False` default argument, which means that "the contents and metadata of the linked files are copied to the new tree" [1]

[1] https://docs.python.org/2/library/shutil.html#shutil.copytree

**Comment by adrianherrera** [comment]
Tested and it works!

**Pull request merged**